server/api/users/me-solves.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../../database')
+const { responses } = require('../../responses')
+
+module.exports = {
+  method: 'get',
+  path: '/users/me/solves',
+  requireAuth: true,
+  handler: async ({ uuid }) => {
+    const userSolves = await db.solves.getSolvesByUserId({ userid: uuid })
+
+    if (userSolves === null) return responses.badUserData
+
+    const returnedSolves = []
+
+    userSolves.forEach(solve => {
```

Comment:
We can probably use `let ret = userSolves.map(solve => solve.challengeid)` and return ret?